content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+
+Halaman ini mengasumsikan kamu menggunakan {{< glossary_tooltip term_id="docker" >}}
+untuk menjalankan Pod, dan Node kamu bejalan menggunakan sistem operasi Fedora.
+Instruksi untuk distribusi lain atau instalasi Kubernetes mungkin beragam.
+
+<!-- steps -->
+
+## Membuat sebuah Pod statis
+
+Kamu dapan mengatur Pod statis dengan menggunakan sebuah
+[berkas konfigurasi pada _file system_](/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-filesystem)
+atau sebuah [berkas konfigurasi ditempatkan pada web](/docs/tasks/configure-pod-container/static-pod/#konfigurasi-via-http).
+
+### Manifes Pod statis pada Filesystem {#konfigurasi-via-filesystem}
+
+Manifes adalah standar definisi Pod dalam format JSON atau YAML pada suatu direktori.
+Gunakan _field_ `staticPodPath: <direktori>` pada
+[berkas konfigurasi kubelet](/docs/tasks/administer-cluster/kubelet-config-file),
+yang akan membaca direktori
+secara berkala dan membuat atau menghapus Pod statis sesuai dengan berkas YAML/JSON
+yang bertambah atau berkurang disana.
+
+Catatan bahwa kubelet akan mengabaikan berkas yang diawali dengan titik (_dot_)
+ketika memindai suatu direktori.
+
+Sebagai contoh, ini cara untuk memulai web server sederhana sebagai Pod statis:
+
+1. Pilih Node yang kamu mau untuk menjalankan Pod statis. Dalam contoh ini adalah `my-node1`.
+
+    ```shell
+    ssh my-node1
+    ```
+
+2. Pilih sebuah direktori, katakan `/etc/kubelet.d` dan letakan berkas definisi Pod untuk web server disana, contohnya `/etc/kubelet.d/static-web.yaml`:
+
+    ```shell
+    # Jalankan perintah ini pada node tempat kubelet sedang berjalan
+    mkdir /etc/kubelet.d/
+    cat <<EOF >/etc/kubelet.d/static-web.yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: static-web
+      labels:
+        role: myrole
+    spec:
+      containers:
+        - name: web
+          image: nginx
+          ports:
+            - name: web
+              containerPort: 80
+              protocol: TCP
+    EOF
+    ```
+
+3. Atur kubelet pada Node untuk menggunakan direktori ini dengan menjalankan menggunakan argumen `--pod-manifest-path=/etc/kubelet.d/`. Untuk Fedora ubah `/etc/kubernetes/kubelet` untuk menambahkan baris berikut:
+
+    ```
+    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --pod-manifest-path=/etc/kubelet.d/"
+    ```
+    atau tambahkan _field_ `staticPodPath: <direktori>` pada [berkas konfigurasi kubelet](/docs/tasks/administer-cluster/kubelet-config-file).
+
+4. Jalankan ulang kubelet. Pada Fedora, kamu dapat menjalankan:
+
+    ```shell
+    # Jalankan perintah berikut pada Node tempat kubelet berjalan
+    systemctl restart kubelet
+    ```
+
+### Manifes Pod statis pada Web {#konfigurasi-via-http}
+
+Kubelet akan secara berkala mengunduh berkas yang ditentukan pada argumen `--manifest-utl=<URL>`
+dan menginterpretasikan sebagai sebuah berkas JSON/YAML yang berisikan definisi Pod.
```

Comment:
Kubelet usually written with lowercase kubelet, we can reorder the sentence for starting with uppercase
```suggestion
Berkas yang ditentukan pada argumen `--manifest-utl=<URL>` akan diunduh oleh kubelet secara berkala
dan kubelet akan menginterpretasikannya sebagai sebuah berkas JSON/YAML yang berisikan definisi Pod.
```